dev_container

```diff
@@ -275,6 +275,10 @@ get_default_img() {
     echo -n "holohub:ngc-v2.0.0-"$(get_host_gpu)
 }
 
+get_compute_capacity() {
```

Comment:
Can we add a comment about what that does?

Reply:
Updated